src/Newsletter.php

```diff
@@ -280,4 +280,52 @@ protected function getSubscriptionOptions(string $email, array $mergeFields, arr
 
         return $options;
     }
+
+    public function getMarketingPermissions(string $listName = ''): array
+    {
+        $list = $this->lists->findByName($listName);
+
+        $response = $this->mailChimp->get("lists/{$list->getId()}/members");
+
+        if (! $this->lastActionSucceeded()) {
+            return false;
```

Comment:
This will throw an exception as the return typehint is `array`

Reply:
Good catch. I removed the typehint.